.evergreen/integration-tests.sh

```diff
@@ -183,3 +183,21 @@ echo $MONGO_SHELL_CONNECTION_FLAGS
 
 `pwd`/mongodb/bin/mongo $MONGO_SHELL_CONNECTION_FLAGS --eval 'printjson(db.serverBuildInfo())' admin
 `pwd`/mongodb/bin/mongo $MONGO_SHELL_CONNECTION_FLAGS --eval 'printjson(db.isMaster())' admin
+
+touch mo-expansion.yml
```

Comment:
Recommend documenting purpose of this `touch` command. Is `mo-expansion.yml` expected to already exist (hence append `>>mo-expansion.yml` below)? If not, it seems more appropriate to remove/replace any existing `mo-expansion.yml` file to avoid the possibility of unexpected additional expansions.

Reply:
`mo-expansion.yml` is expected to not exist before the `touch`. The `touch` was added to limit changing the snippet copied from [run-orchestration.sh from drivers-evergreen-tools](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/651efa00f0118fff65e075706a80f4fafd7b0b04/.evergreen/run-orchestration.sh#L105-L119).

Since `integration-tests.sh` already diverges from run-orchestration, updated to remove the `touch mo-expansion.yml` and change `>>mo-expansion.yml` to `>mo-expansion.yml`.